src/com/company/ShoppingCenter.java

```diff
@@ -0,0 +1,63 @@
+package com.company;
+
+import java.util.LinkedList;
+import java.util.List;
+
+public class ShoppingCenter {
+    private String name;
+    private String address;
+    private List<Shop> shops;
+
+    private int idGenerator = 0;
+
+    public void addShop(Shop shop) {
+        this.shops.add(shop);
+    }
+
+   /* public Shop getShop(int shopId) {
```

Comment:
It seems that the same functionality is done in `findShopById` method. Remove redundant code, please.